app/scripts/scripts.js

```diff
@@ -1,17 +1,28 @@
-/* For picking letters from alphabet keypad. Returns the
-   letter chosen or "already picked"
-*/
-function pickLetter(letter) {
+(function () {
```

Comment:
I openly admit I may be missing something, but this file doesn't seem to be called from index.html.